### DISCUSSION TITLE
Daily 18/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#198] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve viendo como cambiar la parte de registración del front para que utilice las modificaciones de hasheo de contraseña, no pude resolverlo pero sigo trabajando en eso. Tambien realice las modificaciones en el diagrama de clases, está subido a la wiki, falta subirlo como archivo a documentación. Por la tarde participé de la clase sincrónica del ISPC.
Hoy continúo solucionando la conexion del registración con el back. Además voy a seguir revisando documentación y arreglando las activitys para que se visualizen mejor.

**Comment by BraianTroncoso** [comment]
Ayer estuve en clase y hablé con el profesor por los requerimientos del sprint, me reuní con compañeros que necesitaban resolver algunos bugs los cuales pudimos resolver. Hoy hicimos los mismo, vamos avanzando pero encontramos unas fallas, tenemos que resolver algunas cuestiones de instancias de y de registro con los modelos profesional y usuario, también, vamos a priorizar las issues necesarias y resolverlas cuanto antes.
Es mucho trabajo y se complica con todas las responsabilidades que uno tiene. De todas maneras, seguimos avanzando.
